### ISSUE TITLE
Pre-trained weights for CenterPoint_Hierarchy_nuScenes

### ISSUE THREAD

**Issue by weivnaeco** [neeharperi/LT3D#8]
@Redrew @kylevedder @mtli @neeharperi @mayechi 
Hello. 
Can I ask if you can share other pre-trained weights like CenterPoint_Hierarchy_nuScenes used for experiments of paper?
I think only pre-trained weights for CenterPoint_Hierarchy_Argoverse2.0, FCOS3D_Argoverse2.0,  TransFusion-L_Argoverse2.0 are available now.
I hope I can download other pre-trained weights such as CenterPoint_nuScenes(w/, w/o Hierarchy) and  CenterPoint_Argoverse2.0(w/o Hierarchy).
Thank you. Have a nice day.

**Comment by weivnaeco** [comment]
@neeharperi 
Hello. Thank you for replying.
I have 3 more questions.
First, is there any reason why you did not use all 23 classes in nuScenes and  ignore rare classes such as wheelchair and animal (ignore Railed vehicle, Official signaler, Traffic light trailer, Animal in AV2)?
Second, it takes 18 days to train centerpoint_0075voxel_second_secfpn_dcn_4x8_cyclic_50m_wide_hierarchy_tta_20e_nus.py on single NVIDIA GeForce RTX 3090. Do you think it will harm mAP if I increase "samples_per_gpu=1"? 
![image](https://github.com/neeharperi/LT3D/assets/145219037/e1b24d39-cb1d-4f3f-8605-88bbc5e71733)
Last, do you think it will harm mAP if I resume training from latest stopped epoch checkpoint by "'--resume-from'"?
Thank you. Have a good day.

**Comment by neeharperi** [comment]
- We ignore certain classes in nuScenes (e.g. wheelchair) because these classes don't appear  in the validation set. Since AV2 is already a long-tailed dataset, we directly use the official categories. 

- I don't think there will be any harm if you increase samples_per_gpu or resume training from the stopped epoch. I can try to retrain these models on my end once we have a few free GPUs.
